packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -219,38 +239,73 @@ export default class GoogleCalendarService implements Calendar {
       "primary";
 
     try {
-      const event = await calendar.events.insert({
-        calendarId: selectedCalendar,
-        requestBody: payload,
-        conferenceDataVersion: 1,
-        sendUpdates: "none",
-      });
+      let event;
+      let recurringEventId = null;
+      if (calEventRaw.existingRecurringEvent) {
+        recurringEventId = calEventRaw.existingRecurringEvent.recurringEventId;
+        const recurringEventInstances = await calendar.events.instances({
+          calendarId: selectedCalendar,
+          eventId: calEventRaw.existingRecurringEvent.recurringEventId,
+        });
+        event = recurringEventInstances.data.items![calEventRaw.existingRecurringEvent.eventIndex];
+        await calendar.events.patch({
+          calendarId: selectedCalendar,
+          eventId: event.id || "",
+          requestBody: {
+            description: getRichDescription({
+              ...calEventRaw,
+            }),
+          },
+        });
+      } else {
+        const eventResponse = await calendar.events.insert({
+          calendarId: selectedCalendar,
+          requestBody: payload,
+          conferenceDataVersion: 1,
+          sendUpdates: "none",
+        });
+        event = eventResponse.data;
+        if (event.recurrence) {
+          if (event.recurrence.length > 0) {
+            recurringEventId = event.id;
+            const recurringEventInstances = await calendar.events.instances({
+              calendarId: selectedCalendar,
+              eventId: recurringEventId || "",
+            });
+
+            if (recurringEventInstances.data.items) {
+              event = recurringEventInstances.data.items[0];
+            }
+          }
+        }
+      }
 
-      if (event && event.data.id && event.data.hangoutLink) {
+      if (event && event.id && event.hangoutLink) {
         await calendar.events.patch({
           // Update the same event but this time we know the hangout link
           calendarId: selectedCalendar,
-          eventId: event.data.id || "",
+          eventId: event.id || "",
           requestBody: {
             description: getRichDescription({
               ...calEventRaw,
-              additionalInformation: { hangoutLink: event.data.hangoutLink },
+              additionalInformation: { hangoutLink: event.hangoutLink },
             }),
           },
         });
       }
 
       return {
         uid: "",
-        ...event.data,
-        id: event.data.id || "",
+        ...event,
+        id: event.id || "",
+        recurringMeetingId: recurringEventId,
```

Comment:
Better name would be `recurringEventId` only instead of `recurringMeetingId`

Reply:
Renamed it to `thirdPartyRecurringEventId` because there already exists a `recurringEventId` that `calcom` uses